src/BlockchainReader/Bitcoin.ts

```diff
@@ -56,6 +70,10 @@ const dataOutputToBuffer = pipe(
   dataToBuffer
 )
 
+export const isCorrectBufferLength = (buffer: Buffer) => {
+  return buffer.byteLength < anchorByteLength ? false : true
```

Comment:
use implicit return please, and don't believe you need the ternary the expression will already return true false.

Reply:
Updated!